### DISCUSSION TITLE
Should we allows users to configure the min/max number of workers their job uses?

### DISCUSSION THREAD

**Discussion by psschwei** [Qiskit/qiskit-serverless#996] [Ideas]
This value is currently hardcoded when the quantum-serverless instance is created: https://github.com/Qiskit-Extensions/quantum-serverless/blob/v0.6.3/charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml#L196-L197

However, we've had one user ask if it was possible to configure this value for their job. So it might be worth bounds on workers as an additional knob for users to control how their job is run.

**Comment by akihikokuroda** [comment]
This is addressed in issue #1031 and RP https://github.com/Qiskit-Extensions/quantum-serverless/pull/1051
